.github/label-actions.yml

```diff
@@ -1,9 +1,34 @@
 # Configuration for Github App - https://github.com/dessant/label-actions
+#
+# Note: Be aware of the edge cases of YAML when writing multiline strings:
+#   - https://yaml-multiline.info/
+#   - https://github.com/dessant/label-actions/issues/1
 pulls:
   actions:
     needs-docs:
       comment: |
         Thanks for your pull request, before this can be merged - corresponding documentation for your module is required:
+
           - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
           - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
-          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
\ No newline at end of file
+          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
```

Comment:
This one's currently live, example:

![image](https://user-images.githubusercontent.com/60357436/78933354-1b072d80-7aa1-11ea-9009-56fa33f73389.png)